src/components/register/index.tsx

```diff
@@ -1,17 +1,10 @@
-import React from 'react';
+import React, {useCallback} from 'react';
 import {TextField, Box, makeStyles} from '@material-ui/core';
 
 import {BaseButton, WebsiteInfo} from '../common';
 import {PrivatePage} from "../page";
 import {currentStore} from "../../stores";
-
-// TODO: Should move to 0auth library!
-export type DynamicFormInput = {
-  type: string;
-  label: string;
-  required: boolean;
-  name?: string;
-}
+import {defaultValue} from "@0auth/message/lib/type/defaultValue";
```

Comment:
Let's move the library importation to the top of the file.
```js
import React from 'react';
// giving the blank here.
import { PrivatePage } from '../common';
```

I'll fix them when I'm working on #11.